lib/active_publisher.rb

```
@@ -33,6 +34,15 @@ def self.publish(route, payload, exchange_name, options = {})
     end
   end
 
+  def self.publish_all(exchange_name, messages)
+    with_exchange(exchange_name) do |exchange|
+      messages.each do |message|
+        raise "bulk publishing messages must be of type ActivePublisher::Message" unless message.is_a?(ActivePublisher::Message)
```

Perhaps this should raise a specific error?